src/main/java/store/bookshop/exeption/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package store.bookshop.exeption;
```

Comment:
typo in package name
```suggestion
package store.bookshop.exception;
```